openwisp_notifications/widgets.py

```diff
@@ -0,0 +1,39 @@
+from pydoc import locate
+
+from openwisp_notifications import settings as app_settings
+
+
+class IgnoreObjectNotificationWidgetMedia:
+    extend = True
+    js = [
+        'admin/js/jquery.init.js',
+        'openwisp-notifications/js/object-notifications.js',
+    ]
+    css = {'all': ['openwisp-notifications/css/object-notifications.css']}
```

Comment:
could you convert these two lists to tuples for consistency with the other modules please?